src/containers/Review/ReviewSectionRow.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Grid, Message } from 'semantic-ui-react'
+import { Loading } from '../../components'
+import ReviewSectionRowAction from '../../components/Review/ReviewSectionRowAction'
+import ReviewSectionRowAssigned from '../../components/Review/ReviewSectionRowAssigned'
+import ReviewSectionRowLastDate from '../../components/Review/ReviewSectionRowLastAction'
+import ReviewSectionRowProgress from '../../components/Review/ReviewSectionRowProgress'
+import strings from '../../utils/constants'
+import useGetFullReviewStructure from '../../utils/hooks/useGetFullReviewStructure'
+import { AssignmentDetailsNEW, FullStructure, ReviewSectionComponentProps } from '../../utils/types'
+
+// Component renders a line for review assignment within a section
+// to be used in ReviewHome and Expansions
+type ReviewSectionRowProps = {
+  sectionId: number
+  assignment: AssignmentDetailsNEW
+  fullApplicationStructure: FullStructure
+}
+
+const ReviewSectionRow: React.FC<ReviewSectionRowProps> = ({
+  sectionId,
+  assignment,
+  fullApplicationStructure,
+}) => {
+  const { fullReviewStructure, error } = useGetFullReviewStructure({
+    reviewAssignment: assignment,
+    fullApplicationStructure,
+    filteredSectionIds: [sectionId],
+  })
+
+  if (error) return <Message error title={strings.ERROR_APPLICATION_OVERVIEW} list={[error]} />
```

Comment:
```suggestion
  if (error) return <Message error title={strings.ERROR_REVIEW_OVERVIEW} list={[error]} />
```

Or maybe a new one more specific to problem to load a section, since this one will be string: `ERROR_REVIEW_APPLICATION: 'Problem loading application for review'`

Reply:
0146cbebc955116dce30894a8aa0a88b4f5ddfa1
in one of the branches further up, i've changed all of the `ERROR_APPLICATION_OVERVIEW`, used in a few places, to a GENERIC one, i think I would need to change to be more granular. please let me know if this should be done in that PR

Reply:
Oh, nicw! I haven't looked at that one yet. That's good, I agree all other changes to messages should be done in there